<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Syötä nimesi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nimi</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="weighingDateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Finnish" country="Finland"/>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="genderComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Valitse sukupuoli</string>
    </property>
    <item>
     <property name="text">
      <string>Valitse sukupuoli</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mies</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nainen</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mittauspäivä</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pituus (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>70</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="calcPushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>830</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Laske</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>830</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tallenna</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rasvaprosentti (US)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rasvaprosentti (FI)</string>
    </property>
   </widget>
   <widget class="QLabel" name="fatFiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="fatUsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukupuoli</string>
    </property>
   </widget>
   <widget class="QLabel" name="ageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>syntymäaika</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthDateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Finnish" country="Finland"/>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="testUiPushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>870</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Testaa</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>481</width>
      <height>801</height>
     </rect>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pictures/MiesSlice.png);
background-image: url(:/pictures/NaisSlice.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderNeck">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelNeck">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderWaist">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelWaist">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHips">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderHips">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Painoindeksi</string>
    </property>
   </widget>
   <widget class="QLabel" name="bmiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderHeight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>69</y>
      <width>22</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="sliderPosition">
     <number>170</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QDial" name="weightDial">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeight">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>170</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paino (kg)</string>
    </property>
   </widget>
   <zorder>weightDial</zorder>
   <zorder>nameLineEdit</zorder>
   <zorder>nameLabel</zorder>
   <zorder>weighingDateEdit</zorder>
   <zorder>genderComboBox</zorder>
   <zorder>dateLabel</zorder>
   <zorder>heightLabel</zorder>
   <zorder>weightLabel</zorder>
   <zorder>calcPushButton</zorder>
   <zorder>savePushButton</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>fatFiLabel</zorder>
   <zorder>fatUsLabel</zorder>
   <zorder>label</zorder>
   <zorder>ageLabel</zorder>
   <zorder>birthDateEdit</zorder>
   <zorder>testUiPushButton</zorder>
   <zorder>frame</zorder>
   <zorder>horizontalSliderNeck</zorder>
   <zorder>labelNeck</zorder>
   <zorder>horizontalSliderWaist</zorder>
   <zorder>labelWaist</zorder>
   <zorder>labelHips</zorder>
   <zorder>horizontalSliderHips</zorder>
   <zorder>label_11</zorder>
   <zorder>bmiLabel</zorder>
   <zorder>verticalSliderHeight</zorder>
   <zorder>labelHeight</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTiedosto">
    <property name="title">
     <string>Tiedosto</string>
    </property>
    <widget class="QMenu" name="menuJaa">
     <property name="title">
      <string>Jaa</string>
     </property>
     <addaction name="actionFacebook"/>
     <addaction name="actionTwitter"/>
     <addaction name="actionInstagram"/>
     <addaction name="separator"/>
     <addaction name="actionLinkedIn"/>
    </widget>
    <widget class="QMenu" name="menuAsetukset">
     <property name="title">
      <string>Asetukset</string>
     </property>
     <addaction name="actionKieli"/>
     <addaction name="actionUlkoasu"/>
    </widget>
    <addaction name="actionTulosta"/>
    <addaction name="menuJaa"/>
    <addaction name="menuAsetukset"/>
    <addaction name="actionSulje"/>
   </widget>
   <widget class="QMenu" name="menuMuokkaa">
    <property name="title">
     <string>Muokkaa</string>
    </property>
    <addaction name="actionPalauta_oletukset"/>
   </widget>
   <widget class="QMenu" name="menuOhjeet">
    <property name="title">
     <string>Ohjeet</string>
    </property>
    <addaction name="actionOhje"/>
   </widget>
   <addaction name="menuTiedosto"/>
   <addaction name="menuMuokkaa"/>
   <addaction name="menuOhjeet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTulosta">
   <property name="text">
    <string>Tulosta...</string>
   </property>
  </action>
  <action name="actionSulje">
   <property name="text">
    <string>Sulje</string>
   </property>
  </action>
  <action name="actionFacebook">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
  <action name="actionTwitter">
   <property name="text">
    <string>Twitter</string>
   </property>
  </action>
  <action name="actionInstagram">
   <property name="text">
    <string>Instagram</string>
   </property>
  </action>
  <action name="actionLinkedIn">
   <property name="text">
    <string>LinkedIn</string>
   </property>
  </action>
  <action name="actionKieli">
   <property name="text">
    <string>Kieli</string>
   </property>
  </action>
  <action name="actionUlkoasu">
   <property name="text">
    <string>Ulkoasu</string>
   </property>
  </action>
  <action name="actionY_tila">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Yötila</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAutomaattinen_tallennus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automaattinen tallennus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPoista_k_ytt_j">
   <property name="text">
    <string>Poista käyttäjä...</string>
   </property>
  </action>
  <action name="actionPalauta_oletukset">
   <property name="text">
    <string>Palauta oletukset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOhje">
   <property name="text">
    <string>Ohje</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>birthDateEdit</tabstop>
  <tabstop>weighingDateEdit</tabstop>
  <tabstop>genderComboBox</tabstop>
  <tabstop>calcPushButton</tabstop>
  <tabstop>savePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="dialogResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderNeck</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelNeck</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderWaist</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelWaist</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderHips</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelHips</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weightDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weightLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelHeight</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
